### PR TITLE
Next-Gen General Purpose For MI integration

### PR DESCRIPTION
**Pull request by stuti149**
This PR integrates the changes to display Next-Gen General purpose for MI recommendations on UI.

image

### CODE REVIEW

File: extensions/sql-migration/src/constants/strings.ts [hunk 1]
```diff
@@ -332,6 +332,7 @@ export function MI_CONFIGURATION_PREVIEW(hardwareType: string, computeTier: stri
 }
 export const GENERAL_PURPOSE = localize('sql.migration.sku.azureConfiguration.generalPurpose', "General purpose");
 export const BUSINESS_CRITICAL = localize('sql.migration.sku.azureConfiguration.businessCritical', "Business critical");
+export const NEXTGEN_GENERAL_PURPOSE = localize('sql.migration.sku.azureConfiguration.nextGenGeneralPurpose', "Next-Gen General Purpose");
```
Comment (cpichuka) [inline extensions/sql-migration/src/constants/strings.ts:335]: It should be "Next-gen  General Purpose", please check with Ajith once.

Comment (cpichuka) [reply extensions/sql-migration/src/constants/strings.ts:335]: image

**Review by cpichuka — CHANGES_REQUESTED** [review]
Please change the name.

File: extensions/sql-migration/src/constants/strings.ts (hunk 1 above)
Comment (stuti149) [reply extensions/sql-migration/src/constants/strings.ts]: done

**Review by brian-harris — APPROVED** [review]

**Review by cpichuka — APPROVED** [review]

**Pull request merged**